src/runtime_src/core/edge/user/aie/aied.cpp

```diff
@@ -30,31 +30,31 @@
 
 namespace zynqaie {
 
-Aied::Aied(xrt_core::device* device): mCoreDevice(device)
+aied::aied(xrt_core::device* device): m_device(device)
 {
   done = false;
-  pthread_create(&ptid, NULL, &Aied::pollAIE, this);
+  pthread_create(&ptid, NULL, &aied::poll_aie, this);
   pthread_setname_np(ptid, "Graph Status");
 }
 
-Aied::~Aied()
+aied::~aied()
 {
   done = true;
   pthread_kill(ptid, SIGUSR1);
   pthread_join(ptid, NULL);
 }
 
 /* Dummy signal handler for SIGTERM */
-static void signalHandler(int signum) {}
+static void signal_handler(int signum) {}
 
-void* 
-Aied::pollAIE(void* arg)
+void*
+aied::poll_aie(void* arg)
```

Comment:
Can this thread routine communicate with a condition variable and avoid the sleep, along with signal handler?

Reply:
This is old code, just changed the function names and class names to proper ones. Regarding functionality we can work with another PR

Reply:
Nah, std::string constructs from const char* so you don't need the variable.